wolox-books-vue/src/views/NavBar.vue

```diff
@@ -0,0 +1,40 @@
+<template lang="pug">
+  nav.navbar.row.middle.space-around.full-width
+    div.main.row.space-between
+      img.logo(src='../assets/wolox-logo.png')
+      button.bold(@click="logOut") Logout
+</template>
+
+<script>
+import { LocalStorageService } from '../services/localStorageService'
+
+const localStorageService = new LocalStorageService()
```

Comment:
No need of a class here

Reply:
i couldnt find the localStorage file so i used the typescript version which is a class